website/docs/docs/build/build-semantic-layer-intro.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Build your Semantic Layer"
+id: build-semantic-layer-intro
```

Comment:
Can we trim the length here? For consistency, can we come to an agreement between using `sl` and `semantic-layer` in the id? We flip-flop between the two on different pages.

Reply:
yep, good idea! will do